.github/workflows/codeql.yml

```diff
@@ -1,41 +1,41 @@
-name: "CodeQL"
+name: 'CodeQL'
```

Comment:
Run `pnpm format`, something in your IDE settings doesn't factor in the prettier config

Reply:
yes run it manually before commit , thats why this changes are shown